### PR TITLE
Initial addition of terrascan helm chart

### LINKED ISSUE #685: Create Helm charts for the terrascan admission webhook setup.
Would be a much better way to deploy the admission webhook using a helm chart.

### PR DESCRIPTION
**Pull request by jlk**
Fixes #685

### CODE REVIEW

File: deploy/helm-charts/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+keywords:
+  - terrascan
+  - opa
+  - security
+sources:
+  - https://github.com/accurics/terrascan
+maintainers:
+  - name: jlk
+    email: jlk@accurics.com
+  - name: steven-sheehy
```
Comment (dev-gaur) [inline deploy/helm-charts/Chart.yaml:17]: can you remove this name entry?

**Review by rahulchheda — COMMENTED** [review]
/lgtm
nit: let's remove `NOTES.txt`, and `README.md` if needed

**Review by rahulchheda — DISMISSED** [review]

**Comment by dev-gaur** [comment]
Two important points:

1. webhook is missing. you may use something from https://github.com/jlk/terrascan/pull/1

2. the manifests and the helm chart have to be **terrascan compliant** 

Incase the pod the dies down, the webhook will not allow even the new terrascan pod itself to be created. Thus, creating a deadlock.

File: deploy/helm-charts/Chart.yaml (hunk 1 above)
Comment (jlk) [reply deploy/helm-charts/Chart.yaml:17]: :o thx not sure how I left that in!

**Review by kanchwala-yusuf — APPROVED** [review]

**Comment by jlk** [comment]
FYI please don't merge yet - doing a little more work on this, will merge in by COB today.

**Comment by jlk** [comment]
(sidetracked on the homebrew stuff...still chewing on this)

**Comment by jlk** [comment]
Trying to remove myself as a blocker on this. Just pushed some more changes.

@dev-gaur - maybe during your day you can test this out and help with the docs? I haven't made any changes yet around the issue of what happens when the admission controller restarts - probably worth testing by killing the pod and seeing what happens.

File: deploy/helm-charts/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+name: terrascan
+version: 1.0.0
+appVersion: v1.4.0
```
Comment (dev-gaur) [inline deploy/helm-charts/Chart.yaml:4]:
```suggestion
appVersion: v1.5.1
```

File: deploy/helm-charts/values.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+terrascan_api_key: terrakey
+terrascan_container_image: accurics/terrascan:1.5.0
+use_debug: true
+secret_name: terrascancerts
+config_name: terrascanconfig
+name: terrahook
```
Comment (dev-gaur) [inline deploy/helm-charts/values.yaml:6]:
```suggestion
name: terrascan
```
terrahook is a fun name but applicable only if we use the webhook `mode = true`
 .. better to go with the generic `terrascan` now

File: deploy/helm-charts/values.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
+terrascan_api_key: terrakey
+terrascan_container_image: accurics/terrascan:1.5.0
```
Comment (dev-gaur) [inline deploy/helm-charts/values.yaml:2]:
```suggestion
terrascan_container_image: accurics/terrascan:1.5.1
```

**Review by dev-gaur — CHANGES_REQUESTED** [review]

File: deploy/helm-charts/Chart.yaml (hunk 2 above)
Comment (jlk) [reply deploy/helm-charts/Chart.yaml]: bumped version.

File: deploy/helm-charts/values.yaml (hunk 3 above)
Comment (jlk) [reply deploy/helm-charts/values.yaml]: fixed

**Review by dev-gaur — APPROVED** [review]

**Pull request merged**